### ISSUE TITLE
mkcert.exe Insufficient system resources - file gets deleted

### ISSUE THREAD

**Issue by skillshrine** [FiloSottile/mkcert#250]
I'm running in to a very strange issue, whereby suddenly mkcert.exe refuses to run on a developer's PC, where previously it worked fine. I have attached an image of the error and have tried all manner of troubleshooting that I know (created new admin account, running powershell as admin, reinstalled chocolatey, reinstalled mkcert, closed down any programs in the background, rebooted, updated Windows). Literally nothing has helped, and the most bizarre thing is that when we try and run it, and it throws the exception about insufficient system resources, I can watch the file get deleted with my own eyes and appear in the recycle bin.

Any help appreciated.
 
![image](https://user-images.githubusercontent.com/59985273/77374282-62fa3500-6dbe-11ea-8400-5b8293949a96.png)

**Comment by quentinv72** [comment]
I have the same problem. Did you find a way to make it work?

**Comment by skillshrine** [comment]
> 
> 
> I have the same problem. Did you find a way to make it work?

TL;DR - I didn't find a solution, and used another PC.

Hey Quentin, thought I would respond back as I know it's frustrating to find dead-end tickets and the developer appears not to reply.

I never did find a solution I'm afraid, our solution was "Don't use that laptop" and we got the developer's desktop PC from the office instead. I'm a little worried this may happen on other PCs - we were lucky in this case as because of COVID-19 we were working from home, this was a new configuration. The really weird thing was, she had brought her laptop in to the office before we all decided to work from home, and I had setup the machine there. She then used it successfully for the next 2 days. Once we were confident it was all working fine, she took the laptop and started working from home, and then his happened the very next time she used it.

I would love to know if you manage to work it out, sorry I couldn't be of more help.

**Comment by quentinv72** [comment]
Hey, 
Thanks for answering this.
I will keep you updated if I figure this out.

**Comment by bartlangelaan** [comment]
I have the same issue!

I could work around the issue by installing an older version:
```
choco install mkcert --version 1.4.0 --force
```

This is of course not ideal - but it works fine _on my machine_.

**Comment by quentinv72** [comment]
Yes that works for me as well.

**Comment by AstralProgrammer** [comment]
@bartlangelaan fix works on my end. Thanks!

**Comment by FiloSottile** [comment]
Unfortunately I have no experience with Windows internals. Using an older version is of course not a great solution, so if anyone figures it out I would be happy to fix it.
